crates/spk-schema/crates/foundation/src/ident_ops/parsing/request.rs

```diff
@@ -39,3 +41,39 @@ where
         ),
     )(input)
 }
+
+/// Parse a package name with optional components and optional version.
+///
+/// The package name must either be followed by a `/` or the end of input.
```

Comment:
This line seems like it's wrong for this function...

Reply:
It's not wrong but I changed it a little.